src/shared/components/post/post-listing.tsx

```diff
@@ -482,26 +482,13 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
               this.postLink
             )}
           </h5>
-          {(url && isImage(url)) ||
-            (post.thumbnail_url && (
```

Comment:
I think all that's necessary is to remove the or condition. We do want to show an expando for image posts, just not for link posts.